sources/Interop/Windows/winrt/roparameterizediid/Locator.Manual.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from winrt/roparameterizediid.h in the Windows SDK for Windows 10.0.19041.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+namespace TerraFX.Interop
+{
+    public unsafe struct _Locator
+    {
+        void** lpVtbl;
+        delegate*<ushort*, IRoSimpleMetaDataBuilder*, int> _fn;
+
+        public _Locator(delegate*<ushort*, IRoSimpleMetaDataBuilder*, int> fn)
+        {
+            lpVtbl = computedVtbl;
+            _fn = fn;
+        }
+
+        static void** computedVtbl;
```

Comment:
nit: These should all include `private`. Likewise, it would be "nice" to keep the `beforefieldinit` which just involves creating a helper method `private static void** CreateVtbl()`

Reply:
Addressed in #165